docroot/modules/custom/uiowa_core/uiowa_core.module

```diff
@@ -61,6 +61,40 @@ function uiowa_core_form_alter(&$form, FormStateInterface $form_state, $form_id)
       }
       break;
 
+    // @todo Move this to a more appropriate location.
+    case 'node_area_of_study_form':
```

Comment:
I think this will only trigger when creating new area of study content, but not when editing existing forms. But then for editing existing, if adding checkboxes, think will need to check for whether each program type has a value or not.

Reply:
Good point. I only started with the node add form, but this will need to be adapted for the node edit form as well.

Reply:
I removed this, so it is moot now.